### DISCUSSION TITLE
Using regular expressions in CEL Interceptros

### DISCUSSION THREAD

**Discussion by pvlltvk** [tektoncd/triggers#1237] [Q&A] [answered]
Hi everyone!

Is it possible to use regexp in CEL Interceptors  to filter events? For instance, can i use constructions like this:
```
interceptors:
  - cel:
      filter: body.ref == 'tekton/.*'"
```

**Comment by dibyom** [comment, marked as answer]
Hi @pvlltvk sorry for the delay in responding. CEL supports regular expressions using the built in `matches` function. 
(See the `matches` function here: https://github.com/google/cel-spec/blob/master/doc/langdef.md#list-of-standard-definitions)

So, something like `body.ref.matches('tekton/.*')` should work.

**Reply by pvlltvk** [reply]
Hi @dibyom! It works, thanks!
